aria_data_loaders/aria_data_utils/detector_wrappers/object_detector.py

```diff
@@ -149,7 +168,31 @@ def _aria_fetch_demo_heuristics(
 
         return valid, score, stop
 
-    def process_frame(self, img_frame: np.ndarray):
+    def process_frame_online(
```

Comment:
see if we can merge `process_frame_online` with `process_frame_offline`

Reply:
It would be nice to merge the 2.

The "stop" condition is really valuable in on-line case as well as that will reduce compute at run-time

Reply:
Rename process_frame_online to process_frame
Rename process_frame_offline to process_frame_with_stop

Reply:
addressed